### PR TITLE
set dayOfStart input field type to datepicker

### PR DESCRIPTION
**Pull request by AdamVasEpam**
https://trello.com/c/RFdpNBi0

### CODE REVIEW

**Review by kbertalan — CHANGES_REQUESTED** [review]
Please resolve merge conflicts between this branch and master branch

**Comment by AdamVasEpam** [comment]
merge conflicts resolved

File: persistence/src/main/java/com/epam/rft/atsy/persistence/repositories/ApplicationsRepository.java [hunk 1]
```diff
@@ -19,13 +17,4 @@
...
    */
   List<ApplicationEntity> findByCandidateEntity(CandidateEntity candidateEntity);
-
-  /**
-   * Returns a page of ApplicationEntities of the candidate.
-   * @param candidateEntity the given candidate
-   * @param pageable a pageable object
-   * @return a page of ApplicationEntities
-   */
-  Page<ApplicationEntity> findByCandidateEntity(CandidateEntity candidateEntity, Pageable pageable);
```
Comment (kbertalan) [inline persistence/src/main/java/com/epam/rft/atsy/persistence/repositories/ApplicationsRepository.java:29]: Merge of the master branch was not successful.
Features committed into master branch are removed in this branch and accepting this pull request would destroy those functionalities.

**Review by kbertalan — CHANGES_REQUESTED** [review]
Retry the merge.
Please start a new branch from the original solution (first commit in this branch) and merge the changes there.
Also you can delete this branch (and PR) as it became corrupted.

**Pull request closed without merging**